force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -241,8 +311,15 @@ public with sharing class FlowControlLoopsCollections {
         Set<String> stringSet = new Set<String>();
 
         //add strings to the set
+        for (Integer x = 0; x < inputList.size(); x++) {
+            
+            // check if item already exist in the Set and if not add it
+            if (!stringSet.contains(inputList[x])) {
+                stringSet.add(inputList[x]);
+            }
```

Comment:
- No need to check contains. A set, by definition, can not have duplicates. You could add the same value 1000 times it will be there once. 
- Another tip, is that you can pass a list directly into a Set constructor. 

```
  public static Set<String> createStringSet(List<String> inputList) {
        return new Set<String>(inputList); // Replace null with the variable you used to store the result
    }
 ```